ddtrace/appsec/__init__.py

```diff
@@ -19,7 +19,12 @@
 
 def enable(*args, **kwargs):
     if AppSecProcessor is None:
-        log.warning("AppSec was enabled but couldn't start due to the following error:\n%s", _appsec_import_error)
+        # DDAS-001-01
```

Comment:
if these are specific/known error codes, should we be including them in the log message?

they would definitely help support staff if we have a knowledge base of what the error codes mean / how to resolve / etc.